pkg/render/monitor/monitor.go

```diff
@@ -662,6 +668,37 @@ func (mc *monitorComponent) serviceMonitorFluentd() *monitoringv1.ServiceMonitor
 	}
 }
 
+func (mc *monitorComponent) serviceMonitorQueryServer() *monitoringv1.ServiceMonitor {
+	return &monitoringv1.ServiceMonitor{
+		TypeMeta: metav1.TypeMeta{Kind: monitoringv1.ServiceMonitorsKind, APIVersion: MonitoringAPIVersion},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      render.QueryserverServiceName,
+			Namespace: common.TigeraPrometheusNamespace,
+			Labels:    map[string]string{"team": "network-operators"},
+		},
+		Spec: monitoringv1.ServiceMonitorSpec{
+			Selector:          metav1.LabelSelector{MatchLabels: map[string]string{"k8s-app": render.QueryserverServiceName}},
+			NamespaceSelector: monitoringv1.NamespaceSelector{MatchNames: []string{render.QueryserverNamespace}},
+			Endpoints: []monitoringv1.Endpoint{
+				{
+					HonorLabels:     true,
+					Interval:        "5s",
+					Port:            "queryserver",
+					ScrapeTimeout:   "5s",
+					Scheme:          "https",
+					BearerTokenFile: "/var/run/secrets/kubernetes.io/serviceaccount/token",
```

Comment:
I think it would be good to extract a const for this string.

I tried to search for one in our k8s.io dependencies, but was unable to find (to my surprise).

Reply:
Sure. Will do. I did some searches but only hard-coded string or local variables.

Reply:
Extracted.